### PR TITLE
Pop!_Accessibility

### PR DESCRIPTION
**Pull request by dasgeekchannel**
Add some accessibility documentation to Pop!_OS for those who are wanting to install Pop!_OS but may not be familiar with all of the options available from Gnome to assist them from installation onward. Tested these features documented here in latest version of Pop!_OS to make sure they're all current and working.

### CODE REVIEW

**Comment by ahoneybun** [comment]
It looks like there is no title or description in the .md file, the file will need that so that it shows up in the list of documentation.

**Comment by dasgeekchannel** [comment]
> It looks like there is no title or description in the .md file, the file will need that so that it shows up in the list of documentation.

I believe it's updated correctly now. First contribution so appreciate the patience.

File: Pop!_Accessibility.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Accessibility
+description: >
+  Accessibility options for Pop!\_OS
+---
+
+### Pop!\_OS: This document describes some of the applications and utilities that can be used. 
+
+Pop!_OS shares many of the same accessibility options as Gnome. These accessibility options allow for Pop!_OS to be used by as many people in the community as possible. If there are additional tools or options that you would like to see added to Pop!_OS in the future please let us know and we will see if implementation is possible.
```
Comment (brs17) [inline Pop!_Accessibility.md]: GNOME should be capitalized as it is/was an acronym.

Comment (brs17) [inline Pop!_Accessibility.md]: GNOME should be capitalized as it is/was an acronym.

File: Pop!_Accessibility.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Pop!_OS shares many of the same accessibility options as Gnome. These accessibility options allow for Pop!_OS to be used by as many people in the community as possible. If there are additional tools or options that you would like to see added to Pop!_OS in the future please let us know and we will see if implementation is possible. 
+
+Additional information can be found on Gnome accessibility web page [here](https://help.gnome.org/users/gnome-help/stable/a11y.html).
+
+## Screen Readers
+For visual impairments [Orca](https://help.gnome.org/users/orca/stable/introduction.html.en) is a screen reading tool for low sighted and blind users.
+
+*Orca is a free, open source, flexible, and extensible screen reader that provides access to the graphical desktop via speech and refreshable braille.* 
+
+In Pop!_OS using the Gnome desktop environment, Orca can be launched automatically via the Universal Access Control Center under the Seeing section.
```
Comment (brs17) [inline Pop!_Accessibility.md]: `under the Settings section` (Settings was spelled Seeing)

**Comment by brs17** [comment]
@ahoneybun is this file (Pop!_Accessibility.md) in the right place in this repository? Does it need to be in docs/docs?

**Comment by ahoneybun** [comment]
@brs17 Ah yes the .md file will need to be the docs folder for it to appear on the website.

**Comment by dasgeekchannel** [comment]
> @brs17 Ah yes the .md file will need to be the docs folder for it to appear on the website.

Is there something I need to do here to move it?

**Comment by ahoneybun** [comment]
Yes you need to move the .md file into the docs folder for it to show up when we merge it in.

**Comment by ahoneybun** [comment]
LGTM! Thank you!

**Comment by brs17** [comment]
https://pop.system76.com/docs/Pop!_Accessibility/ :tada:

**Pull request merged**